server/routes/article-route.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = function(controllers, middlewares) {
```

Comment:
__[eslint]__
* Error - 'module' is not defined. (no-undef)
* Error - 'middlewares' is defined but never used (no-unused-vars)